ci-operator/config/openshift/cluster-operator/master.json

```diff
@@ -0,0 +1,89 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "root": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    },
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    },
+    "ansible": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "ansible"
+    },
+    "test_base_image": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    }
+  },
+  "canonical_go_repository": "github.com/openshift/cluster-operator",
+  "binary_build_commands": "NO_DOCKER=1 make build",
+  "images": [
+    {
+      "from": "base",
+      "to": "cluster-operator",
+      "dockerfile_path": "build/cluster-operator/Dockerfile",
+      "inputs": {
+        "bin": { "paths": [{ "source_path": "/go/src/github.com/openshift/cluster-operator/bin/cluster-operator", "destination_dir": "." }] }
+      }
+    },
+    {
+      "from": "ansible",
+      "to": "cluster-operator-ansible",
+      "context_dir": "build/cluster-operator-ansible"
+    },
+    {
+      "from": "root",
+      "to": "builder-base",
+      "dockerfile_path": "build/build-image/Dockerfile"
+    },
+    {
+      "from": "builder-base",
+      "to": "test-base",
+      "dockerfile_path": "build/test-image/Dockerfile"
+    }
+  ],
+  "tests": [
+    {
+      "as": "verify",
+      "from": "test-base",
+      "commands": "git reset HEAD --hard && git clean -f -d && NO_DOCKER=1 make verify"
```

Comment:
What's going on here?

Reply:
During verification we use git to determine if code has been generated properly. The problem is that when the test is run the git tree is dirty, partly because of a replaced FROM in one of the image files. The reset and clean just make sure we start off with a clean tree.

Reply:
Is this going to mess up Origin commit verifies, etc? We should strive to give you a "good" checkout -- can we fix something upstream?

Reply:
Seems like this is a problem with the test-base step?

Reply:
It is